packages/dashboard/src/components/elements/account-wizard/containers/address-details-form/address-details.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { AddressDetails } from '@deriv/account';
+import { useStores } from 'Stores';
+
+const AddressDetailsForm: React.FC = props => {
+    const { client_store } = useStores();
+    const { states_list, fetchStatesList, residence } = client_store;
```

Comment:
FYI, for functions it's recommended not to destructure if they reference `this` as it'll remove the `this` context.